src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir-Book;
+  font-style: normal;
+  font-weight: 300;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-style: normal;
+  font-weight: 900;
+  src: url(/fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
+  margin: 0;
+}
```

Comment:
* selector is for box-sizing: border-box; 
and let's reset all margin and padding like you did on the lesson